### DISCUSSION TITLE
build-armbian里面的dtb和内核里面的dtb有什么区别呢？

### DISCUSSION THREAD

**Discussion by bigmangos** [ophub/amlogic-s9xxx-armbian#2708] [Rockchip] [answered]
build-armbian里面的dtb
build-armbian/armbian-files/platform-files/rockchip/bootfs/dtb/rockchip

编译内核里面也有设备对应的dtb
实际使用的是哪个呢？

**Comment by ophub** [comment, marked as answer]
实际用的是内核的，在打包脚本里可以看到代码，解压内核并覆盖同名文件了。

仓库的为人看的，方便查阅有哪些可用的dtb，方便测试新设备时找同家族的其他文件，更改/boot/uEnv.txt时好写

**Reply by bigmangos** [reply]
追问一下大佬，每次内核版本提升，我看了都要更新config文件，整个问题是怎么来的呢

**Reply by greatofdream** [reply]
> 实际用的是内核的，在打包脚本里可以看到代码，解压内核并覆盖同名文件了。
> 
> 仓库的为人看的，方便查阅有哪些可用的dtb，方便测试新设备时找同家族的其他文件，更改/boot/uEnv.txt时好写

替换的命令是[这里](https://github.com/ophub/amlogic-s9xxx-armbian/blob/main/rebuild#L831C32-L831C61)吗？

**Comment by ophub** [comment]
对，整个replace_kernel() 就是替换几个文件的。
